src/components/SectionTitle/SectionTitle.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { styles } from "../../styles";
+import { makeStyles, Typography } from "@material-ui/core";
+import IconMaker from "../IconMaker/IconMaker";
+
+const useStyles = makeStyles(theme => ({
+  titleCont: {
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+    height: styles.sizes.globalDimensions.percentageFull
+  },
+  titleStyle: {
+    fontFamily: "serif"
+  },
+  blue: {
+    color: styles.colors.lightBlue
+  }
+}));
+
+const SectionTitle = ({ iconImage, title, family, weight, size }) => {
```

Comment:
wrap this props as they are styles into an object and then down below just do a spread operator

Reply:
if they are different styles for different elemeents then group them into respectives objects